meta-luneos/classes/webos_app_generate_security_files.bbclass

```diff
@@ -23,13 +23,23 @@ def webos_app_generate_security_files_write_permission_file(d, app_info):
     key    = app_id + "-*"
     type   = app_info["type"]
 
+    # START Hack for legacy com.palm apps to fix issues with proxied LS2 calls, we want to make sure to allow the "app_id*" and "app_id-*"
+    key2 = app_id
+    # END Hack for legacy com.palm apps to fix issues with proxied LS2 calls, we want to make sure to allow the "app_id*" and "app_id-*"
+
     if type == "native":
         key = app_id
 
     permission = {}
 
     if "requiredPermissions" in app_info:
         permission[key] = app_info["requiredPermissions"]
+        
+        # START Hack for legacy com.palm apps to fix issues with proxied LS2 calls, we want to make sure to allow the "app_id*" and "app_id-*"
+        if type == "web" and "com.palm." in app_id:
+            permission[key2] = app_info["requiredPermissions"]
```

Comment:
I think it would be clearer if we just use permission [app_id] and no key2 intermediate variable